assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/clients/DataCiteClient.java

```diff
@@ -56,37 +54,35 @@
     public static final String ERROR_SETTING_DOI_URL_TEMPLATE =
         ERROR_SETTING_DOI_URL
             + DOI_AND_HTTP_STATUS_TEMPLATE_ENTRIES;
+
+    private static final String HTTP_STATUS_LOG_MESSAGE = "{}";
     public static final String ERROR_CREATING_DOI_TEMPLATE =
         ERROR_CREATING_DOI
             + PREFIX_TEMPLATE_ENTRY
-            + HTTP_STATUS_LOG_TEMPLATE;
+            + HTTP_STATUS_LOG_TEMPLATE
+            + HTTP_STATUS_LOG_MESSAGE;
     private static final Logger logger = LoggerFactory.getLogger(DataCiteClient.class);
-    private final DataCiteMdsConnectionFactory mdsConnectionFactory;
+    private final DataCiteConnectionFactory dataCiteApiConnectionFactory;
```

Comment:
renaming classes since it produces connections both to MDS and to REST API of DataCite